kratos/python/add_quaternion_to_python.cpp

```diff
@@ -48,6 +49,10 @@ void  AddQuaternionToPython(pybind11::module& m) {
     .def_property("Y", QuaternionGetY, QuaternionSetY)
     .def_property("Z", QuaternionGetZ, QuaternionSetZ)
     .def_property("W", QuaternionGetW, QuaternionSetW)
+    .def("Normalize",        &Quaternion<double>::normalize)
+    .def_static("Identity",  &Quaternion<double>::Identity)
+    .def("ToRotationMatrix", &Quaternion<double>::ToRotationMatrix<Matrix>)
+    .def("MultiplyQuaternion", MultiplyQuaternion)
```

Comment:
If you use a lambda then you don't need the aux function

```suggestion
    .def("MultiplyQuaternion", [](Quaternion<double>& self, Quaternion<double>& a, Quaternion<double>& b){return Quaternion<double>(operator*(a,b));})
```

BTW why does this have three arguments and not two? Can you show the usage?

Reply:
How can I use the lambda function in this case?
3 arguments are needed as 2 input quaternions for the multiplication and the first argument is the quaternion which calls the function. I'm calling the function like that in my code:
self._quaternion = self._quaternion.MultiplyQuaternion(self._delta_quaternion, self._quaternion)

Reply:
ok I understand
I checked the c++ code it is consistent